### PR TITLE
DG v1.4 Preliminary Check

### LINKED ISSUE #147: Screen through DG

### PR DESCRIPTION
**Pull request by Jaylenlee**
Note the following will need to be updated:

Model component -> ModelClassUDiagram (Add UniqueTagList and Tag to diagram)

Model component -> BetterModelClassDiagram (rename address book to expense book)

### CODE REVIEW

**Comment by maxxyh** [comment]
> Note the following will need to be updated:
> 
> Model component -> ModelClassUDiagram (Add UniqueTagList and Tag to diagram)
> 
> Model component -> BetterModelClassDiagram (rename address book to expense book)

Sorry @Jaylenlee I don't really understand do we need to make these new diagrams?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -378,30 +303,30 @@ Step 3. The user can now use the following command to trigger a FindCommand.
 * **Alternative 2:** Allows aliases of all command words. Does not override default command words but merely adds an alias. Reserved keywords cannot be applied unless it is for its associated subclass (i.e. removing the custom alias).
   * Pros: More flexibility than Alternative 1.
   * Cons: Restricts degree of customisation due to reserved keywords not being allowed to use as alias for other Commands.
-  
+
 * **Alternative 3:** Allows customisation of ALL command words.
   * Pros: Highest degree of flexibility, better for users who can easily get used to Command Line Apps.
   * Cons: May be messy and slower learning users may get confused.
-_{more aspects and alternatives to be added}_
+_{more aspects and alternatives to be added}
```
Comment (lebencwb) [inline docs/DeveloperGuide.md]: I think there may be a formatting error here as the phrase is intended to be italicized.

**Review by lebencwb — CHANGES_REQUESTED** [review]
Thanks for updating the DG, @Jaylenlee! Most of it looks good to me - just one formatting issue to address and it should be good to go!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Jaylenlee) [reply docs/DeveloperGuide.md]: Thanks for the note! I have fixed it.

**Review by lebencwb — APPROVED** [review]
LGTM!

**Pull request merged**